### ISSUE TITLE
Most efficient material trades not being recommended

### ISSUE THREAD

**Issue by Repsycl3d** [msarilar/EDEngineer#538]
For example, I am sitting on a mountain of pharmaceutical isolators but it's instead telling me to trade 850% of my core dynamics composites instead of 3 pharmaceutical isolators (not listed at all by the popup) to get the 17 chemical manipulators. This is happening only to some of my material trade recommendations. I need heat vanes and it wants me to trade all my rare wrong category tier 5s instead of the pile of proto heat radiators i already have.. etc

**Comment by msarilar** [comment]
is it possible you need the pharmaceutical isolators in your shopping list?

**Comment by Repsycl3d** [comment]
No, they weren't required. I traded and got what I needed, and even though
EDengineer showed them as unobtainable with the alternate color, I obtained
them anyway by trading down unsuggested materials. Didn't always have this
problem, and no idea what's gone wrong.

**Comment by alterNERDtive** [comment]
Did it report the correct amount of PI in your materials list?
